tests.edn

```diff
@@ -0,0 +1 @@
+#kaocha/v1 {:reporter kaocha.report/documentation}
```

Comment:
I like it here for the overview you get for failing tests. It works well for that because there are just a few tests. In datahike there are so many test though that it gets less clear with the verbosity.